operations/availabilityblocks.md

```diff
@@ -39,10 +39,15 @@ Returns all availability blocks filtered by services, unique identifiers and oth
     "ExternalIdentifiers": [
         "Block-0001"
     ],
+    "ActivityStates": [
+        "Deleted",
```

Comment:
by default we might include just the `Active` here , so the people are not overusing the `Deleted` states and use it if and only if they really need it.